web/templates/settings/user_topics_settings.hbs

```diff
@@ -1,14 +1,27 @@
 <div id="user-topic-settings" class="settings-section" data-name="topics">
     <p>
-        {{#tr}}
-            Zulip lets you mute topics and streams to avoid receiving notifications messages you are not interested in.
-            Muting a stream effectively mutes all topics in that stream. You can also manually mute a topic in an unmuted stream,
-            or unmute a topic in a muted stream. <z-link>Learn more.</z-link>
-            {{#*inline "z-link"}}<a href="/help/mute-a-topic" target="_blank" rel="noopener noreferrer">{{> @partial-block}}</a>{{/inline}}
-        {{/tr}}
+        {{#if development}}
+            {{#tr}}
+                Zulip lets you configure notifications for each topic. You can also automatically follow or unmute topics
+                you start or participate in.
+            {{/tr}}
```

Comment:
This copy probably isn't what we'll want long-term; linking to /help/ documentation seems valuable, and we're also not exactly configuring notifications directly; we're configuring notifications policy for the topic at a somewhat higher level...